### PR TITLE
Fix googlenet

### PR DESCRIPTION
**Pull request by sakaia**
GoogLeNet model outputs 3 arguments.
This fix takes first tensor (logits).

Pull Request resuolved: #559

### CODE REVIEW

**Comment by soumith** [comment]
googlenet is weird enough, i think we shouldn't support it in examples/imagenet/main.py 
the network doesn't converge when trained from scratch either
